S107: Update examples for C# and VB.NET (#2209)
SonarSource/rspec: rules/S107/vbnet/rule.adoc

[source,vbnet]
----
' Each function does a part of what the original setCoordinates function was doing, so confusion risks are lower
Sub SetOrigin(x As Integer, y As Integer, z As Integer)
   ' ...
End Sub

Sub SetSize(width As Integer, height As Integer, depth As Integer)
   ' ...
End Sub
----

* Find a better data structure for the parameters

[source,vbnet]
----

> Class Point ' Point is a concept example that logically groups data'

Reviewer: ```suggestion
Class Point ' In geometry, Point is a logical structure to group data
```

Reviewer: I don't have a strong opinion regarding Class vs Structure in VB. Keep it a mutable class here as turning this into a properly defined Structure requires too much code.
